metabase/metabase
# Do not Move and Archive personal collections

Fixes https://github.com/metabase/metabase/issues/28127

How to verify:
- Open the root collection. You should be able to see a personal collection of another user
- Open the menu for it. It should not be possible to Move or Archive it. 

<img width="778" alt="Screenshot 2023-02-14 at 14 27 24" src="https://user-images.githubusercontent.com/8542534/218738622-127b92ad-2a4d-4956-bf80-0ad6ff1303a1.png">
<img width="761" alt="Screenshot 2023-02-14 at 14 27 31" src="https://user-images.githubusercontent.com/8542534/218738630-3645a6cb-d817-4351-8a5f-45caa83c8878.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/28229)
